cfgrammar/src/lib/yacc/parser.rs

```diff
@@ -462,19 +462,23 @@ impl YaccParser {
 
         // Next, the '(' pattern : type, ... ')'
         i = self.parse_ws(i, false)?;
-        if let Some(mut j) = self.lookahead_is("(", i) {
+        if let Some(_) = self.lookahead_is("(", i) {
```

Comment:
Can be `if self.lookahead_is(...).is_some()`?

Reply:
good catch!